content/training/Walk-through/place_analyses.Rmd

```diff
@@ -174,12 +177,12 @@ ggplot(ar, aes(x = reorder(Region, `AR (per 10,000)`),
 
 You could also **table mortality rate by region**. A few things to note:  
 
-* The first command uses `group_by()` to count cases by `patient_origin`. Then, `filter()` is applied based on the character search function `grepl()` from the package **stringr**, which looks for "Dead" within the variable `exit_status` - **note: we must change this to reflect our data and have `grepl()` look for "Died" in `exit_status2`** (otherwise, the object `deaths` has 0 rows and `mortality_rate()` produces an error).  
+* The first command uses `group_by()` to count cases by `patient_origin`. Then, `filter()` is applied based on the character search function `grepl()` from the package **stringr**, which looks for "Dead" within the variable `exit_status` - **note: we must change this to reflect our data and have `grepl()` look in `exit_status2`** (otherwise, the object `deaths` has 0 rows and `mortality_rate()` produces an error).  
```

Comment:
I've changed all grep functions to str_ functions in the templates

```suggestion
* The first command uses `group_by()` to count cases by `patient_origin`. Then, `filter()` is applied based on the character search function `str_detect()` from the package **stringr**, which looks for "Dead" within the variable `exit_status` - **note: we must change this to reflect our data and have `str_detect()` look in `exit_status2`** (otherwise, the object `deaths` has 0 rows and `mortality_rate()` produces an error).  
```